tests/installation/setup_online_repos.pm

```diff
@@ -55,7 +61,10 @@ sub run() {
 
     send_key $cmd{next};    # Next
 
-    assert_screen "desktop-selection", 200;    # Make sure repos setup is finished and went to next step already
+    if (get_var("WITH_MAIN_REPO")) {
```

Comment:
yet another variable with a very generic name and a very specific meaning - we urgently need a variable naming convention ;(